test/e2e/node/pod_resize.go

```diff
@@ -0,0 +1,1076 @@
+				}
+			}
+			for i := range tc.containers {
+				setDefaultPolicy(&tc.containers[i])
+			}
+			for i := range tc.expected {
+				setDefaultPolicy(&tc.expected[i])
+			}
+
+			tStamp := strconv.Itoa(time.Now().Nanosecond())
+			tPod = makeTestPod(ns, "testpod", tStamp, tc.containers)
+
+			ginkgo.By("creating pod")
+			pod := podClient.CreateSync(tPod)
+
+			ginkgo.By("verifying the pod is in kubernetes")
+			selector := labels.SelectorFromSet(labels.Set(map[string]string{"time": tStamp}))
+			options := metav1.ListOptions{LabelSelector: selector.String()}
+			pods, err := podClient.List(context.TODO(), options)
+			framework.ExpectNoError(err, "failed to query for pods")
+			gomega.Expect(len(pods.Items) == 1)
+
+			ginkgo.By("verifying initial pod resources, allocations, and policy are as expected")
+			verifyPodResources(pod, tc.containers)
+			verifyPodResizePolicy(pod, tc.containers)
+
+			ginkgo.By("verifying initial pod status resources and cgroup config are as expected")
+			verifyPodStatusResources(pod, tc.containers)
+			verifyPodContainersCgroupConfig(pod, tc.containers)
+
+			ginkgo.By("patching pod for resize")
+			pPod, pErr = f.ClientSet.CoreV1().Pods(pod.Namespace).Patch(context.TODO(), pod.Name,
+				types.StrategicMergePatchType, []byte(tc.patchString), metav1.PatchOptions{})
+			framework.ExpectNoError(pErr, "failed to patch pod for resize")
+
+			ginkgo.By("verifying pod patched for resize")
+			verifyPodResources(pPod, tc.expected)
+			verifyPodAllocations(pPod, tc.containers)
+
+			ginkgo.By("verifying pod spec resources equal pod status resources after resize")
+			waitPodStatusResourcesEqualSpecResources := func() (*v1.Pod, error) {
+				for start := time.Now(); time.Since(start) < PollTimeout; time.Sleep(PollInterval) {
+					pod, err := podClient.Get(context.TODO(), pod.Name, metav1.GetOptions{})
+					if err != nil {
+						return nil, err
+					}
+					differs := false
+					for idx, c := range pod.Spec.Containers {
+						if diff.ObjectDiff(c.Resources, *pod.Status.ContainerStatuses[idx].Resources) != "" {
+							differs = true
+							break
+						}
+					}
+					if differs {
+						continue
+					}
+					return pod, nil
+				}
+				return nil, fmt.Errorf("timed out waiting for pod spec resources to match status resources")
+			}
+			rPod, rErr := waitPodStatusResourcesEqualSpecResources()
+			framework.ExpectNoError(rErr, "failed to get pod")
+
+			ginkgo.By("verifying pod container's cgroup configs, pod resources, pod allocations, and pod status after resize")
+			verifyPodContainersCgroupConfig(pPod, tc.expected)
+			verifyPodResources(rPod, tc.expected)
+			verifyPodAllocations(rPod, tc.expected)
+			verifyPodStatusResources(rPod, tc.expected)
+
+			ginkgo.By("deleting pod")
+			err = e2epod.DeletePodWithWait(f.ClientSet, pod)
+			framework.ExpectNoError(err, "failed to delete pod")
+		})
+	}
+}
+
+var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling] [Slow]", func() {
```

Comment:
This won't run as part of the regular test suite with the `[Slow]` selector so we're unable to see the results of running these tests against CI. Can you please remove that for now so this can run as part of the all alpha job?

```suggestion
var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
```

I'd like to see the results reported through https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/102884/pull-kubernetes-e2e-gce-alpha-features/1457140410781536256/

Reply:
@ehashman , thanks, I will revise accordingly.